src/frontend/apps/desk/src/features/addMembers/components/ModalAddMembers.tsx

```diff
@@ -54,8 +54,12 @@ export const ModalAddMembers = ({
   const [selectedMembers, setSelectedMembers] = useState<OptionsSelect>([]);
   const [selectedRole, setSelectedRole] = useState<Role>(Role.MEMBER);
   const { toast } = useToastProvider();
-  const { mutateAsync: createInvitation } = useCreateInvitation();
-  const { mutateAsync: createTeamAccess } = useCreateTeamAccess();
+  const { mutateAsync: createInvitation, isPending: isPendingInvitation } =
+    useCreateInvitation();
+  const { mutateAsync: createTeamAccess, isPending: isPendingTeamAccess } =
+    useCreateTeamAccess();
```

Comment:
I don't think it is the best way to do, because createInvitation or createTeamAccess can be called multiple times, so if you have multiple members to add, I guess isPending will switch from true to false.

I would set isPending to true here, before the promise stuff:
https://github.com/numerique-gouv/people/blob/b355acb80ae0c69c744799a29cfe6fb3c75ebd9c/src/frontend/apps/desk/src/features/addMembers/components/ModalAddMembers.tsx#L118

And set to false before the onClose:
https://github.com/numerique-gouv/people/blob/b355acb80ae0c69c744799a29cfe6fb3c75ebd9c/src/frontend/apps/desk/src/features/addMembers/components/ModalAddMembers.tsx#L121-L122